### PR TITLE
cuda and mt working, changed input_image read and write

### PR DESCRIPTION
**Pull request by Ugu16**
needed to change input_image read and write to accommodate complex text files for inverse, see @664 on piazza

### CODE REVIEW

File: src/Multi_Threaded/threads_dft.cc [hunk 1]
```diff
@@ -56,28 +55,20 @@ void mt_dft_2d(int argc, char **argv, bool is_reverse) {
...
-		img_transpose[col + row * img_width].real /= img_width * img_height;
-		img_transpose[col + row * img_width].imag /= img_width * img_height;
-	    }
-	}
-    }
-    
-    image_handler.save_image_data(argv[3], img_transpose, img_width, img_height);
-    delete [] img_transpose;
+	if(is_reverse)
+    	image_handler.save_image_data_real(argv[3], img_transpose, img_width, img_height);
```
Comment (Jeblas) [inline src/Multi_Threaded/threads_dft.cc:67]: Leave out the if else statement since both execute the same code

**Review by Jeblas — APPROVED** [review]
May still need to add enum for forward and reverse selection for Cuda

File: src/Multi_Threaded/threads_dft.cc (hunk 1 above)
Comment (Ugu16) [reply src/Multi_Threaded/threads_dft.cc:67]: one write reals, one writes in complex, see pull request description

Comment (Jeblas) [reply src/Multi_Threaded/threads_dft.cc:67]: Gotcha

**Pull request merged**